example/example.h

```diff
@@ -0,0 +1,15 @@
+#ifndef GROUP_EXAMPLE_HEADER_GUARD
+#define GROUP_EXAMPLE_HEADER_GUARD
+
+#include <stdint.h>
+#include <stddef.h>
+
+
+void set_function_index_group_example(void** functions);
```

Comment:
I'd recommend writing `void **functions` instead of `void** functions`. The former reads like C code while the latter indicates C++ style. See also my other comment suggesting `void *functions[]`.